types/react-collapse/index.d.ts

```diff
@@ -0,0 +1,28 @@
+// Type definitions for react-collapse 4.0
+// Project: https://github.com/nkbt/react-collapse
+// Definitions by: Adam Binford <https://github.com/Kimahriman>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+import * as React from 'react';
+
+export interface CollapseProps extends React.HTMLProps<Collapse> {
+    isOpened: boolean;
+    children: React.ReactNode;
+
+    springConfig?: { [key: string]: number }; // react-motion doesn't export the config interface
```

Comment:
> react-motion doesn't export the config interface

If I'm not mistaken, it exports it here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6e3c284a47b9e9aeb925e6488e8825bc13808057/types/react-motion/index.d.ts#L11

Reply:
Ah, I read that type wrong 😬 

What works for me is:

```
import { OpaqueConfig } from "react-motion"
// ...
springConfig?: OpaqueConfig
```

but I'm only passing react-motion presets to react-collapse.

Reply:
Yeah neither are quite exactly right. I wasn't sure whether to redefine `SpringHelperConfig` completely or not, and just decided not to.

Reply:
Aha. The problem I ran into is that passing react-motion presets to react-collapse gave me a type error...

(Thanks for making the type definition though, by the way!)